### ISSUE TITLE
System output selfContained support

### ISSUE THREAD

**Issue by guybedford** [6to5/6to5#633]
I've just been integrating 6to5 into the SystemJS builder project, and it would be quite nice when generating a bundle to use the selfContained transform so that helpers are only defined once per bundle.

When I enable the transformer though, I get the following error:

```
TypeError: file:/Users/guybedford/Projects/systemjs-builder/test/tree/first.js: Cannot set property '_importSource' of undefined
    at SystemFormatter._addImportSource (/Users/guybedford/Projects/6to5/lib/6to5/transformation/modules/system.js:23:22)
    at SystemFormatter.importSpecifier (/Users/guybedford/Projects/6to5/lib/6to5/transformation/modules/system.js:64:8)
    at File.addImport (/Users/guybedford/Projects/6to5/lib/6to5/file.js:256:26)
    at /Users/guybedford/Projects/6to5/lib/6to5/transformation/transformers/other/self-contained.js:66:19
    at File.get (/Users/guybedford/Projects/6to5/lib/6to5/file.js:239:28)
    at File.addHelper (/Users/guybedford/Projects/6to5/lib/6to5/file.js:276:22)
    at ReplaceSupers.superProperty (/Users/guybedford/Projects/6to5/lib/6to5/transformation/helpers/replace-supers.js:40:15)
    at ReplaceSupers.specHandle (/Users/guybedford/Projects/6to5/lib/6to5/transformation/helpers/replace-supers.js:234:28)
    at visitor.enter (/Users/guybedford/Projects/6to5/lib/6to5/transformation/helpers/replace-supers.js:127:21)
    at TraversalContext.enterNode (/Users/guybedford/Projects/6to5/lib/6to5/traverse/index.js:65:16)
```

Any tips on how I could go about setting this up would be appreciated.

**Comment by sebmck** [comment]
Any chance I can get the source for `/Users/guybedford/Projects/systemjs-builder/test/tree/first.js`?

**Comment by guybedford** [comment]
Sure - the source I'm testing is the following:

```javascript
import 'jquery-cdn';
import '@empty';
import { dep } from "./second";
import './amd';
import './component.jsx!./jsx';

export var p = 5;
export class test {
  constructor() {
    super();
  }
}
```

**Comment by guybedford** [comment]
Nice!! Thanks so much for the quick response.

**Comment by sebmck** [comment]
No problem! Fixed as of 3.0.16, thanks!
